### ISSUE TITLE
add option that only forces wrapping if a trailing comma is manually entered

### ISSUE THREAD

**Issue by electrovir** [electrovir/prettier-plugin-multiline-arrays#4]
Potential option name: multilineArraysOnTrailingComma. Would not work in JSON because JSON doesn't support trailing commas.

For example, with this option turned on, this would stay on one line:
```typescript
const thing = ['a', 'b'];
```
whereas this would wrap:
```typescript
const thing = ['a', 'b',];
```

**Comment by monim67** [comment]
This would really be a great feature, then we can opt in for multi-line where we need it to without any comments.

**Comment by monim67** [comment]
Is it going to support function parameters and arguments?

```js
function greet(
  param0,
  param1,
  param2,
) {}

greet(
  'arg 0',
  'arg 1',
  'arg 2',
);
```

**Comment by electrovir** [comment]
@monim67 

> Is it going to support function parameters and arguments?

This is currently not supported. I created a new issue for that feature: #5

**Comment by electrovir** [comment]
This feature has been released in version [`1.0.0`](https://www.npmjs.com/package/prettier-plugin-multiline-arrays)! Note that there are other changes include din that version as well, some noted at the top of the README: `v1.0 breaking changes`.
